packages/smart-wallet/test/test-psm-integration.js

```diff
@@ -153,7 +154,8 @@ test('want stable', async t => {
   t.is(purseBalance(computedState, stableBrand), swapSize - 1n);
 });
 
-test('govern offerFilter', async t => {
+// FIXME this broke after https://github.com/Agoric/agoric-sdk/pull/6166
```

Comment:
please create an issue and assign to me.  (Unless you want to fix it in this PR.)